apps/omg_watcher/lib/event.ex

```diff
@@ -66,14 +62,13 @@ defmodule OMG.Watcher.Event do
     Notifies about invalid block
     """
 
-    def name, do: :invalid_block
-
-    defstruct [:hash, :number, :error_type]
+    defstruct [:hash, :number, :error_type, name: :invalid_block]
 
     @type t :: %__MODULE__{
             hash: Block.block_hash_t(),
             number: integer(),
-            error_type: atom()
+            error_type: atom(),
+            name: atom()
```

Comment:
why some Event have name, and some haven't?

Reply:
Name have events from *pull model*, these returned from `/status`. We fight with `channel` weirdness.